src/main/java/io/pravega/connectors/flink/table/catalog/pravega/descriptors/PravegaCatalogDescriptor.java

```diff
@@ -57,24 +57,37 @@ public PravegaCatalogDescriptor(String controllerUri, String schemaRegistryUri,
     protected Map<String, String> toCatalogProperties() {
         final DescriptorProperties properties = new DescriptorProperties();
 
+        // Pravega controller URI
```

Comment:
We can put these comments where the constants are created which can give us a clear view.